### ISSUE TITLE
Maxbots ports?

### ISSUE THREAD

**Issue by regionalradio** [fgsfdsfgs/perfect_dark#566]
Any chance of a maxbots pc (then switch) port? As combat simulator is a massive part of the game being able to have a fully customisable version would be amazing and running natively would give a performance boost for more sims

**Comment by fgsfdsfgs** [comment]
`maxbots` has a lot of code changes. Some of its features could be implemented in the port, but a direct port would only be possible on a separate branch or something, not in mainline port.

**Comment by regionalradio** [comment]
Would that ever be in the pipeline? Combat simulator is basically it's own game (and I'll yell that it's better than single player forever lol) so having massive deathmatches would be amazing

**Comment by fgsfdsfgs** [comment]
I don't really have a pipeline. Something like raising the sim cap in combat might be a good idea, but I don't even know what else maxbots has.

**Comment by regionalradio** [comment]
The biggest thing apart from more sims was stripping away unnecessary code (like all of single player) to increase memory for better performance, but would that even be necessary now?

**Comment by ludacriswins** [comment]
I agree being able to increase the sim count would be a great addition

**Comment by FrostKnight** [comment]
A thought occured to me on this, would it be possible to have at most 12 players? Like, 2 people 10 bots, for example.

And it will throw an error if you try to put 3 people and 10 bots, or well you know what I mean.

And the error will prevent you from starting till its fixed.

Would the game be able to handle that ideal? I rarely ever have 4 players even in the past. lol

**Comment by LonelySpaceDetective** [comment]
> A thought occured to me on this, would it be possible to have at most 12 players? Like, 2 people 10 bots, for example.
> 
> And it will throw an error if you try to put 3 people and 10 bots, or well you know what I mean.
> 
> And the error will prevent you from starting till its fixed.
> 
> Would the game be able to handle that ideal? I rarely ever have 4 players even in the past. lol

Depends on how human players and bots are handled differently from each other, I'd guess.
Can't exactly speak from authority here, but the problem might be that PD may be designed such that 8 bots is just the hard maximum, and even reducing the number of players doesn't make a difference with how the memory is laid out.
